.github/renovate.json5

```diff
@@ -0,0 +1,46 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "dependencyDashboard": true,
+  "suppressNotifications": ["prEditedNotification"],
+  "extends": ["config:base"],
+  "labels": ["internal"],
+  "schedule": ["before 4am on Monday"],
+  "separateMajorMinor": false,
+  "enabledManagers": [
+    "github-actions",
+    "pre-commit",
+    "cargo",
+    "pep621",
+    "npm",
+  ],
+  "pep621": {
+    "fileMatch": [
+      "^(python|scripts)/.*pyproject\\.toml$",
+    ],
+  },
+  "npm": {
+    "fileMatch": [
+      "^playground/.*package\\.json$",
+    ],
+  },
+  "pre-commit": {
+    "enabled": true,
+  },
+  "ignoreDeps": [
+    // The latest versions of these are not compatible with our release workflow
+    "actions/upload-artifact",
+    "actions/download-artifact",
+  ],
```

Comment:
Can we still doing things like `@<bot> ignore this version`?

Reply:
I don't think you can `@` renovate in the same way you can with dependabot. It's possible to add config to this file specifying that it should permanently ignore certain versions (or version ranges), however. Additionally, if you close a renovate PR, it will generally assume that you don't want to see any future PRs updating the dependency to that version. Docs here: https://docs.renovatebot.com/key-concepts/pull-requests/#normal-prs